app/src/main/java/com/codingblocks/cbonlineapp/viewmodels/MyCourseViewModel.kt

```diff
@@ -22,8 +22,8 @@ import com.codingblocks.onlineapi.Clients
 class MyCourseViewModel(
     var courseDao: CourseDao,
     var runDao: CourseRunDao,
-    private var sectionWithContentsDao: SectionWithContentsDao,
-    private var contentsDao: ContentDao,
+    var sectionWithContentsDao: SectionWithContentsDao,
```

Comment:
why remove private from them

Reply:
because I did not take the mutable approach. I'm updating it as well as all the viewmodels to follow this

Reply:
because I passed them as parameters into the adapter, let me find the perfect way out